### DISCUSSION TITLE
Implement MAPE metric?

### DISCUSSION THREAD

**Discussion by cdeterman** [online-ml/river#1187] [General]
I see that we can use MAE in the metrics of the river models in the examples and the API docs.  However, I don’t see MAPE anywhere.  Would there be a way to implement that metric that I am missing?

**Comment by MaxHalford** [comment]
I just added it.

**Reply by cdeterman** [reply]
Wonderful, I assume I would need to install directly from GitHub to use it then?

**Reply by MaxHalford** [reply]
Yep, you need to install the development version:

```sh
pip install git+https://github.com/online-ml/river --upgrade
```
